modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -422,16 +422,18 @@ CV__DNN_INLINE_NS_BEGIN
     class CV_EXPORTS QuantizeLayer : public Layer
     {
     public:
-        float scale;
-        int zeropoint;
+        bool is1D;
+        std::vector<float> scales;
+        std::vector<int> zeropoints;
```

Comment:
Replace `scale` with `scales`, so that we can support the `scales` of 1-D tensor. And this change effects the ABI compatibility.

Reply:
> ABI compatibility

API?

Reply:
more details can be found in the un-passed ci.